File: js/instantJazz.js
// Problem# 168
// Create a function which concantenates the number 7 to the end of every chord in an array.
// Ignore all chords which already end with 7.
// https://edabit.com/challenge/EMuszDzF66k9J73HG

const jazzify = (arr) =>
  arr.map((chord) => (chord.endsWith("7") ? chord : `${chord}7`));

module.exports = jazzify;

File: python/instant_jazz.py
"""
Instant JAZZ

Create a function which concantenates the number 7 to the end of every chord in a list. Ignore all chords which already end with 7.

https://edabit.com/challenge/jhghtvT2s58FnDr5T
"""
from typing import List


def jazzify(lst: List[str]) -> List[str]:
    return [f"{chord}7" if not chord.endswith('7') else chord for chord in lst]
